connector_x/src/main.cpp

```diff
@@ -347,6 +347,22 @@ void requestEvent()
         break;
     }
 
+    case CommandType::GetColor:
+    {
+        auto runner = getPatternRunner(ledPort);
+        uint32_t msg = runner->getCurrentColor();
+
+        res.responseData.responseReadColor.msg = msg;
+        break;
+    }
+
+    case CommandType::GetPort:
+    {
+        uint8_t msg = ledPort;
+        res.responseData.responseReadPort.msg = msg;
```

Comment:
nit: assign `ledPort` directly to `res.responseData.responseReadPort.msg`

Reply:
fixed